google-cloud-resource_manager/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+### 0.31.0 / 2019-02-12
+
+* Add parent resoure to `Project`:
+  * Add `Project#parent`.
+  * Add `project` optional named argument to `Manager#create_project`.
```

Comment:
Typo: The name of the argument is `parent`.

https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud-resource_manager/lib/google/cloud/resource_manager/manager.rb#L203

Reply:
Great catch. Thanks. Fixed.